PasscodeLock/src/main/java/org/wordpress/passcodelock/AbstractPasscodeKeyboardActivity.java

```diff
@@ -24,9 +25,11 @@ public abstract class AbstractPasscodeKeyboardActivity extends Activity {
     protected InputFilter[] filters = null;
     protected TextView topMessage = null;
 
+	@SuppressLint("RestrictedApi")
```

Comment:
Added this to avoid linter complaining for now, but tracking separately here https://github.com/Automattic/simplenote-android/issues/1734